src/main/java/edu/kis/powp/jobs2d/drivers/adapter/TransformationAdapter.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.jobs2d.features.Transformations.TransformationDecorator;
+
+public class TransformationAdapter implements Job2dDriver {
+    private Job2dDriver driver;
+    private String name;
+    private TransformationDecorator transformator;
```

Comment:
Does it really need to be TransformationDecorator or Transformation would be enough?